meinberlin/apps/contrib/templates/meinberlin_contrib/item_detail.html

```diff
@@ -121,14 +119,14 @@ <h2 class="moderator-statement__title">{% trans 'Official Statement' %}</h2>
                         {{ object.moderator_statement.statement | safe }}
                     </div>
                 </div>
-            </div>
         {% endif %}
     {% endblock %}
```

Comment:
The endblock needs to be more indented to match the opening block tag. Maybe the whole block after the opening thing should be (never sure about the indentation with the block tags).